<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWindow</class>
 <widget class="QDialog" name="ExportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Export settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="vertLayoutSettings">
     <item>
      <widget class="QLabel" name="lblExportType">
       <property name="text">
        <string>Choose export type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radBtnClipboard">
       <property name="text">
        <string>Clipboard</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radBtnCppFile">
       <property name="text">
        <string>.cpp file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radBtnCppAndHFile">
       <property name="text">
        <string>.h and .cpp files</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutFileSettings">
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditFolderPath"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFileName"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblFileName">
         <property name="text">
          <string>File name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFolderPath">
         <property name="text">
          <string>Folder path:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horLayoutButtons">
     <item>
      <widget class="QPushButton" name="pshBtnAccept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pshBtnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
